tests/test_queuing.py

```diff
@@ -319,29 +319,10 @@ def test_update_error(self):
     def test_append_annotating_object(self):
         """Test appending an object that writes annotations when queuing itself"""
 
-        class AnnotatingTensor(qml.operation.Tensor):
```

Comment:
No longer needed, due to the existence of `Tensor.queue()`.

Note: the fact that we needed a dummy class here, and couldn't use the existing Tensor, should have been a sign something was wrong!

Reply:
The removal of this was enabled by the addition of the `queue` method in the regular `Tensor` class above, right?

Reply:
yep!